src/main/resources/application.yaml

```diff
@@ -121,6 +121,15 @@ darts:
         transcription_available: a928742c-5eb7-46f0-9f61-d89d5394785c
         transcription_request_approved: 29c26992-df77-4ba6-8ba7-03324eb5ae37
         transcription_request_rejected: 739a31cf-13a1-49bc-bcf9-0794f4670dbb
+  storage:
+    blob:
+      connection-string: ${AZURE_STORAGE_CONNECTION_STRING:}
```

Comment:
I think this will default to an empty string if the env var is not set. Is that really what we want though? 

Instead, consider removing the `:`, such that the application will fail to start (fail fast) if the env var is not set, i.e:
```yaml
connection-string: ${AZURE_STORAGE_CONNECTION_STRING}
```